files/quora-widget/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "quora-widget",
+  "repo": "arpitbbhyani/quora-widget",
+  "files": {
+    "include": ["quora-widget.js", "quora-widget.min.js"]
```

Comment:
@jimaek  I have removed the entry of `quora-widget.js`. Thanks for pointing that out.